MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -0,0 +1,212 @@
+#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+#import <UserNotifications/UserNotifications.h>
+#endif
+
+#import "MSDeviceTracker.h"
+#import "MSMobileCenterInternal.h"
+#import "MSPush.h"
+#import "MSPushInternal.h"
+#import "MSPushLog.h"
+#import "MSPushPrivate.h"
+
+/**
+ * Service storage key name.
+ */
+static NSString *const kMSServiceName = @"Push";
+
+/**
+ * Key for storing push token
+ */
+static NSString *const kMSPushServiceStorageKey = @"kmspushservicepushstoringkey";
```

Comment:
Doesn't need to have `kms` as prefix of the string. Either `@"PushServiceStorageKey"` or `@"pushServiceStorageKey"` should be fine.